src/include/btmem.h

```diff
@@ -858,7 +858,8 @@ struct __wt_page_deleted {
     WT_UPDATE **update_list; /* List of updates for abort */
 };
 
-/* WT_TIME_PAIR --
+/*
+ * WT_TIME_PAIR --
```

Comment:
We no longer use `WT_TIEM_PAIR` anywhere, it can be removed.

Reply:
`WT_TIME_PAIR` is currently used in many functions in `pm-1521-...` branch, so I left it as it is.